src/nativeTrackerManager.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Script to handle firing impression and click trackers from native teamplates
```

Comment:
Can you convert this to factory pattern just like used in renderingManager and env. It will make writing test cases easier. Also add some unit tests.